DagPut broken in 1.2.0

I just upgraded this repo to 1.2.0 to fix the `AddDir` bug and now:

```
package main

import (
	"log"

	"github.com/ipfs/go-ipfs-api"
)

func main() {
	s := shell.NewLocalShell()
	obj, err := s.DagPut(`{"x": "abc","y":"def"}`, "json", "cbor")
	if err != nil {
		log.Panic(err)
	}
	log.Println(obj)
}
```

```
2017/11/23 22:41:31 dag/put: File argument 'object data' is required
panic: dag/put: File argument 'object data' is required
```

I haven't replaced the daemon. Still running the same version

--- a/dag.go
+++ b/dag.go
@@ -9,7 +9,7 @@ import (
 	"io/ioutil"
 	"strings"
 
-	files "github.com/whyrusleeping/go-multipart-files"
+	files "github.com/ipfs/go-ipfs-cmdkit/files"
 )
 
 func (s *Shell) DagGet(ref string, out interface{}) error {

--- a/shell_test.go
+++ b/shell_test.go
@@ -233,3 +233,12 @@ func TestObjectStat(t *testing.T) {
 	is.Equal(stat.LinksSize, 3)
 	is.Equal(stat.CumulativeSize, 1688)
 }
+
+func TestDagPut(t *testing.T) {
+	is := is.New(t)
+	s := NewShell(shellUrl)
+
+	c, err := s.DagPut(`{"x": "abc","y":"def"}`, "json", "cbor")
+	is.Nil(err)
+	is.Equal(c, "zdpuAt47YjE9XTgSxUBkiYCbmnktKajQNheQBGASHj3FfYf8M")
+}

